src/Uno.Toolkit.UI/Extensions/RectExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Foundation;
+
+namespace Uno.Toolkit.UI
+{
+	internal static class RectExtensions
+	{
+		public static bool IsEmptyOrZero(this Rect? rect) => IsEmptyOrZero(rect.GetValueOrDefault());
```

Comment:
the extension should not dictate the null behavior; at least judging by the name
the caller should be explicit about it: `?.IsEmptyOrZero() ?? true/false`